### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by JCDavie** [BloodHoundAD/BloodHound#223]
It would be nice to have a release roadmap of some sorts.  I would like to contribute but its hard to know the "todo" features without any direction.

**Comment by rvazarkar** [comment]
I would love to support a roadmap, and I'll to work on one when I get a chance.

The unfortunate answer right now is that we have no idea what the roadmap looks like just yet, so it's hard to provide one =(

**Comment by jesseflorig** [comment]
@JCDavie I agree re: a roadmap. However, in the interim I just decided to refactor a component. There's definitely low hanging fruit scattered throughout the repo.

@rvazarkar Ever thought about opening up a [Trello](https://trello.com/) board? It's great for just  rack & stacking ideas.
